microsoft-power-bi/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,43 @@
+= Microsoft Power BI Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft Power BI (Power BI Connector) synchronizes data and automates business processes between Power BI and third-party applications, either on-premises or in the cloud. Power BI is a suite of analytics tools designed to visualize data from various systems and garner insights. 
+
+For compatibility information and fixed issues, see the Microsoft Power BI Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Microsoft Power BI APIs
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access the target resource
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Connector
+
+Common use cases for Power BI Connector include the ability to automate business processes by integrating Power BI with other IT tools and systems in your organization, including:
+
+* Optimizing sales operations by analyzing your sales data (opportunities, leads, and so on) by feeding data from your CRM system to Power BI, and then using Power BI to visualize patterns and identify trends. 
+* Getting a 360-degree view of your customer by feeding data streams from all customer data stores (CRM, support, marketing, and so on) to Power BI and using Power BI to see a customer-centric view.
```

Comment:
I wonder if "a 360-degree view" is colloquial?

Reply:
Hhhhmmmmm good question. For now I'll leave it and do a little research. It's mathematical (loosely) so maybe understood.